contracts/pool/ConcentratedLiquidityPool.sol

```diff
@@ -723,17 +724,12 @@ contract ConcentratedLiquidityPool is IConcentratedLiquidityPoolStruct, IPoolFac
         int128 amount
     ) internal {
         Position storage position = positions[owner][lower][upper];
-
         (uint256 rangeFeeGrowth0, uint256 rangeFeeGrowth1) = rangeFeeGrowth(lower, upper);
 
-        uint256 amount0Fees;
-        if (rangeFeeGrowth0 > position.feeGrowthInside0Last) {
-            amount0Fees = FullMath.mulDiv(rangeFeeGrowth0 - position.feeGrowthInside0Last, position.liquidity, FixedPoint.Q128);
-        }
-
-        uint256 amount1Fees;
-        if (rangeFeeGrowth1 > position.feeGrowthInside1Last) {
-            amount1Fees = FullMath.mulDiv(rangeFeeGrowth1 - position.feeGrowthInside1Last, position.liquidity, FixedPoint.Q128);
+        unchecked {
```

Comment:
add unchecked statement

reference : ConcentratedLiquidityPool: rangeFeeGrowth and secondsPerLiquidity math needs to be unchecked code-423n4/2021-09-sushitrident-2-findings#13